main.tf

```diff
@@ -321,5 +353,25 @@ resource "aws_cloudformation_stack_set" "lacework_stackset" {
   permission_model = "SERVICE_MANAGED"
   template_url     = "https://s3.amazonaws.com/${var.cf_s3_bucket}/${var.cf_s3_prefix}/templates/lacework-aws-cfg-member.template.yml"
 
-  depends_on = [aws_lambda_function.lacework_setup_function]
+
+  depends_on = [ // depending on all this ensures the stackinstances can be torn down
+    aws_s3_bucket.lacework_org_lambda,
+    aws_sns_topic.lacework_sns_topic,
+    aws_sns_topic_subscription.lacework_sns_subscription,
+    aws_sns_topic_policy.default,
+    aws_lambda_permission.lacework_lambda_permission,
+    aws_secretsmanager_secret.lacework_api_credentials,
+    aws_lambda_function.lacework_setup_function
+  ]
+}
+
+
+data "aws_region" "current" {}
+resource "aws_cloudformation_stack_set_instance" "lacework_stackset_instances" {
+  deployment_targets {
+    organizational_unit_ids = [var.organization_unit]
```

Comment:
Is this only 1 org unit? or can this be multiple ones?

Reply:
Can be multiple! I've left like this ATM to match expectations of the generate command. If we want to make this an array (totally fine) we need to update generate. 

Can do now or raise a separate issue.